cmd/admin-handlers.go

```diff
@@ -284,6 +284,73 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	writeSuccessResponseJSON(w, jsonBytes)
 }
 
+// ServerDrivePerfInfo holds informantion about address and write speed of all drives in a single server node
+type ServerDrivePerfInfo struct {
+	Addr       string            `json:"addr"`
+	DrivesPerf []DriveWriteSpeed `json:"drivesPerf"`
+}
+
+// DriveWriteSpeed holds informantion about write speed of a single drive
+type DriveWriteSpeed struct {
+	Path       string  `json:"path"`
```

Comment:
This should be called Disk instead of Path, perhaps also its endpoint representation? such that we know which server the disk belongs to.